Tribler/Test/test_as_server.py

```diff
@@ -114,16 +114,8 @@ def setUpFileServer(self, port, path):
         factory = Site(resource)
         self.file_server = reactor.listenTCP(port, factory)
 
-    @blocking_call_on_reactor_thread
     @inlineCallbacks
     def checkReactor(self, phase, *_):
-        delayed_calls = reactor.getDelayedCalls()
```

Comment:
Just to be sure, this was removed because trial does this automatically right?

Reply:
yes